syncoid

```diff
@@ -379,7 +384,7 @@ sub syncdataset {
 		# and because this will ony resume the receive to the next
 		# snapshot, do a normal sync after that
 		if (defined($receivetoken)) {
-		    my $sendcmd = "$sourcesudocmd $zfscmd send -t $receivetoken";
+		    my $sendcmd = "$sourcesudocmd $zfscmd send $sendoptions -t $receivetoken";
```

Comment:
this won't work mit for example -Lec as zfs -t only supports -e out of those flags.

Reply:
So just read through the zfs code, quite interesting. [Did you know there is an undocumented "raw" send available with -w? I certainly didn't ].

So zfs send -t -Lce would work just fine, because the resume token contains the flags from the original send and they will override any set with -t. -p would be rejected by the zfs utility if set with -t. 

However it clearly isn't correct (or sensible). I can remove the override from the zfs send -t path in syncoid to fix that properly. And allow -p in the original send, which will break right now.

Reply:
@mattaw yeah, but the raw send feature isn't in a stable release yet. It will be released together with the encryption feature in the upcoming 0.8 release to make secure remote backups possible.

You are right, despite what the manpag says, that combinations works.